Makefile

```diff
@@ -33,9 +33,17 @@ else # GNU/Linux, at least (Windows should probably use cmake)
 endif
 
 # installation directories (for 'make install')
-prefix=/usr/local
-libdir=$(prefix)/lib
-includedir=$(prefix)/include
+DESTDIR     ?=
+# directory variables : GNU conventions prefer lowercase
+# see https://www.gnu.org/prep/standards/html_node/Makefile-Conventions.html
+# support both lower and uppercase (BSD), use uppercase in script
```

Comment:
You introduced the uppercase variables, but didn't actually use them in the `install` target below

Reply:
Better have only the lowercase variants, things get really messy with two variables for each directory.